src/forms/ToggleSwitch.jsx

```diff
@@ -1,105 +1,28 @@
 import PropTypes from 'prop-types';
 import React from 'react';
-import cx from 'classnames';
 
-import {
-	C_TEXT_PRIMARY,
-	C_TEXT_SECONDARY,
-	C_TEXT_HINT,
-} from 'swarm-constants/dist/js/constants.js';
-
-import Button from './Button';
-import Icon from '../media/Icon';
-import ToggleWrapper from '../utils/components/ToggleWrapper';
-
-export const TOGGLE_SWITCH_CLASS = 'toggleSwitch';
-export const TOGGLE_SWITCH_ACTIVE_CLASS = `${TOGGLE_SWITCH_CLASS}--active`;
-export const TOGGLE_SWITCH_DISABLED_CLASS = `${TOGGLE_SWITCH_CLASS}--disabled`;
-
-export const KNOB_CLASS = `${TOGGLE_SWITCH_CLASS}-knob`;
-export const KNOB_ACTIVE_CLASS = `${KNOB_CLASS}--active`;
-export const KNOB_DISABLED_CLASS = `${KNOB_CLASS}--disabled`;
-
-export const LABEL_CLASS = `${TOGGLE_SWITCH_CLASS}-label`;
+import { Toggle } from '@meetup/swarm-components';
 
 /**
  * @module ToggleSwitch
  */
 class ToggleSwitch extends React.Component {
 	render() {
-		const {
-			className,
-			disabled,
-			isActive,
-			label,
-			labelClassName,
-			labelledBy,
-			name,
-			onCallback,
-			offCallback,
-			...other
-		} = this.props;
-
-		const labelId = labelledBy || `label-${name}`;
+		const { isActive, label, labelledBy, id, name, ...other } = this.props;
 
-		const getActiveIconColor = isActive =>
-			isActive ? C_TEXT_PRIMARY : C_TEXT_SECONDARY;
-		const getColor = (disbled, isActive) =>
-			disabled ? C_TEXT_HINT : getActiveIconColor(isActive);
+		const labelProps = labelledBy ? { ariaLabelledby: labelledBy } : {};
```

Comment:
The `araiLabelledby` prop should be `'aria-labelledby'` since all `aria-*` attributes are fully supported in jsx.